### PR TITLE
ESBuild dev server

### PR DESCRIPTION
**Pull request by erikmejias**
Using ESBuild serve API instead of custom packages.

### CODE REVIEW

File: bin/build.js [hunk 1]
```diff
@@ -1,36 +1,44 @@
...
-  server.listen(3000, () => console.log('Serving at http://localhost:3000'));
+  esbuild
+    .serve(
+      {
+        servedir: buildDirectory,
+        port: 3000,
+      },
+      {
+        ...defaultSettings,
+        entryPoints: entryPoints,
```
Comment (alexiglesias93) [inline bin/build.js:38]: This can be simplified to just `entryPoints`

**Review by alexiglesias93 — CHANGES_REQUESTED** [review]
Nice! Just a mini tweak, but it looks good!

File: bin/build.js (hunk 1 above)
Comment (erikmejias) [reply bin/build.js]: Oops! 😃

**Comment by alexiglesias93** [comment]
@erikmejias Can we do some further refactoring? b49fa3d269e6fda6f81d943397f49b43b2d34c31

**Pull request merged**